modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -384,18 +384,17 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
 	has_digitigrade = TRUE
 	gender = MALE
 	use_static = TRUE
+/datum/sprite_accessory/underwear/panties_basic
+	name = "Panties - Basic"
```

Comment:
Why are you renaming them? That's just bound to cause issues.

Reply:
I understand the intent behind this change, but unless you include a migration script for those that were using these, I feel like it's only bound to cause issues for players.